cloudinary-image-management-and-manipulation-in-the-cloud-cdn/php/class-sync.php

```diff
@@ -234,7 +234,7 @@ public function can_sync( $attachment_id, $type = 'file' ) {
 	public function get_sync_version( $attachment_id ) {
 		$version = $this->managers['media']->get_post_meta( $attachment_id, self::META_KEYS['plugin_version'], true );
 
-		return $version;
+		return $version !== $this->plugin->version;
```

Comment:
This is always gonna return a bool now. We should update the docblock and update the method name to something like `check_sync_version`. The current name doesn't really apply anymore 😄